pandas/core/arrays/boolean.py

```diff
@@ -378,3 +378,12 @@ def _logical_method(self, other, op):
 
         # i.e. BooleanArray
         return self._maybe_mask_result(result, mask)
+
+    def _accumulate(
+        self, name: str, *, skipna: bool = True, **kwargs
+    ) -> BaseMaskedArray:
+        from pandas.core.arrays import IntegerArray
+
+        data = self._data.astype(int)
```

Comment:
So we're deviating behavior for min/max compared to numpy?

```
In [4]: np.cumprod(aarr)
Out[4]: array([1, 0, 0])

In [5]: np.maximum.accumulate(aarr)
Out[5]: array([ True,  True,  True])

In [6]: np.cumsum(aarr)
Out[6]: array([1, 1, 2])

In [7]: np.minimum.accumulate(aarr)
Out[7]: array([ True, False, False])
```

Reply:
No not on purpose. Fixed this